### DISCUSSION TITLE
Copy and Paste from Word and Bytedeck text features harmony?

### DISCUSSION THREAD

**Discussion by mattjmoore** [bytedeck/bytedeck#1673] [Q&A]
I am just throwing this out there to anyone user or creator...

When creating Quests, does anyone have any experience with how well copy and paste from Word works with the text features that are built into Bytedeck?  in other words, if I were to create quests as interactive assignment documents in Word, would it be easy to just copy and paste from them directly into Bytedeck and maintain all the text features like bolding, italics, etc.?

**Comment by tylerecouture** [comment]
The basic formatting should work (bold, italics, font-size, headings, bullets etc).   What do you mean by "interactive" documents?

**Comment by mattjmoore** [comment]
Oh I just meant a document that acts like the quest page in Bytedeck for a topic.  I was picturing a traditional note taking framework with some interactive elements like links or applet thingys that make the notes a bit more interactive.  I heard a specific term years ago for this type of lesson organization (as opposed to making a powerpoint to present, note sheets, worksheets with questions for after, you just put it all in one interactive document) in a course I was taking, but now I can't remember exactly what they called it.  Your "quest" pages in Bytedeck are essentially exactly what I mean by an interactive lesson document.

Matt

**Comment by mattjmoore** [comment]
The main point of this question is really to ask how I can save all the material I make to use in Bytedeck.  If I make a whole course in Bytedeck and then want to not use Bytedeck at some point in the future, would I lose all the materials if I hadn't also made a word copy of each quest or something along those lines?

**Comment by tylerecouture** [comment]
You can bulk export your quests into a CSV file (which will include the html markup, and all the settings for the quest).  But if you don't want to rely on that, you would need to have the content saved somewhere else.

If Bytedeck is your primary use of the content (for now), then I'd suggest creating the content in Bytedeck directly, and then copy pasting it into an archive (word doc or google doc) if you want that kind of back up.

If you're comfortable with HTML, the most reliable would be to copy and paste the HTML into a backup file, via codeview in the editor:
![image](https://github.com/user-attachments/assets/4d5e215c-d3de-41a2-92ff-edfa15e40ae7)

**Comment by mattjmoore** [comment]
Ok, excellent, that totally makes sense.  Thanks.
